tests/test_utils.py

```diff
@@ -61,7 +61,7 @@ def test_get_extended_access_token():
 def test_get_extended_access_token_v23_plus():
     mock_request.return_value.status_code = 200
     mock_request.return_value.content = (
-        '{"access_token":"<extended access token>","token_type":"bearer"}'
+        '{"access_token":"<extended access token>","token_type":"bearer", "expires_in":5170982}'
```

Comment:
I may be missing something, but I'm not getting a key called `expires_in` from this endpoint in Graph API versions 2.3 and above:

```http
GET /v2.3/oauth/access_token?grant_type=fb_exchange_token&client_id=xxx&client_secret=xxx&fb_exchange_token=xxx HTTP/1.1
Host: graph.facebook.com
Connection: close


HTTP/1.1 200 OK
Access-Control-Allow-Origin: *
Pragma: no-cache
Cache-Control: private, no-cache, no-store, must-revalidate
facebook-api-version: v2.3
Expires: Sat, 01 Jan 2000 00:00:00 GMT
Content-Type: application/json; charset=UTF-8
x-fb-trace-id: EdS3FRiHsz8
x-fb-rev: 2728374
Vary: Accept-Encoding
X-FB-Debug: AQi1j55HPsiRM/ClUEaoHh1vXLdi6knogi/0Dm4XPVQn5ofy543pXk0uEMNrYufp1cr3jsyvJp/QYyO/8phW/A==
Date: Thu, 08 Dec 2016 14:24:38 GMT
Connection: close

{"access_token":"EAAG6WgWqL5cBAIuOwC2TPCJzA3jEypMKvNwu0ZBPZAwZA0oZB91znZBCAJjZBtGlorVpSQWVj9sb2oG7OwDIYX8h6wuIbFLcfpZCnInW73KeOjqcZAx4K0QL4RyZBcX2twgZBZCMEPJZCzy4TGDbLP8YT6Dr","token_type":"bearer"}
```

Do you know why you're getting it, @irkho?